modules/backend/models/brandsetting/custom.less

```diff
@@ -81,7 +89,7 @@ table.table.data {
     tbody {
         tr.active td {
             &:first-child {
-                border-left: 3px solid @brand-secondary;
+                border-left-color: @brand-secondary;
```

Comment:
@	gergo85 why did you remove `3px border`?

Reply:
The custom.less file only contains color definition, is not?

Reply:
Mainly - yes, but just removing the border. If it's a part of the ticket, please provide before/after example!

There is a logo section beside colors.
```
//
// Logos
//

.oc-logo-transparent when not (@logo-image = '') {
    background-image: url('@{logo-image}') !important;
}

.oc-logo when not (@logo-image = '') {
    background-image: url('@{logo-image}');
}
```